code/datums/gamemode/factions/legacy_cult/trash_machinery.dm

```diff
@@ -1,93 +0,0 @@
-//Machinery that gets deleted by Nar-Sie and isn't replaced with cult structures
```

Comment:
is this all moved elsewhere?

Reply:
@Zth-- pls respond

Reply:
This is a duplicated file. You can find the other one here https://github.com/vgstation-coders/vgstation13/blob/817b0d8b685e195454e5311b0cbe3e8b52d1af36/code/datums/gamemode/factions/bloodcult/bloodcult_cultify_exceptions.dm